### PR TITLE
Adding SMS Sender Script

### LINKED ISSUE #60: SMS Sending Script Using Kalyra API
SMS Sending Script Using Kalyra API

This is a python script that uses the kaleyra API for sending SMS. Kaleyra is one of the leading SMS service provider in the industry

Running the script provides users different kind of options for sending SMS. You can choose anyone and proceed with their SMS marketing campaign. This uses the Kaleyra API and make a nice CLI tool

## Do you want to work on this?
- [x] yes
- [ ] no

### PR DESCRIPTION
**Pull request by advaita-saha**
# Description

Added the script for sending SMS using the Kaleyra API

Fixes #60 

## Adding the Script for SMS sending and the readme file for further guidance

- [x] New Script (non-breaking change which adds a new script to the repository)

## Has This Been Tested?

- [x] I confirm that I have tested my script locally and verified that it works as intended.

## Checklist:

- [x] I have gone through the contributing guidelines before making this Pull Request.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: scripts/SMS Sender/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# SMS Sender 
+Easy CLI interface for users looking to test their SMS campaign integration with Kaleyra 
+
+The requirements for any user are :-
+* A kaleyra account and api key, secret key
```
Comment (OBITORASU) [inline scripts/SMS Sender/README.md:5]: Please document how an user can generate their own api key and use it in the script. Use this [documentation](https://github.com/GDSC-RCCIIT/General-Purpose-Scripts/blob/main/scripts/Vaccination-Centre-Locator/README.md) for reference lets say.

**Review by OBITORASU — CHANGES_REQUESTED** [review]
Left a review.

**Comment by OBITORASU** [comment]
Also don't forget to edit the main readme and add your script to the available script table.

**Comment by advaita-saha** [comment]
@OBITORASU I have made the required changes in the documentation. Please the changes

**Comment by OBITORASU** [comment]
You need to resolve the merge conflicts before we can merge this PR.

**Review by OBITORASU — APPROVED** [review]
LGTM!

**Comment by belelaritra** [comment]
@advaita-saha Please change the broken link from [here](https://github.com/GDSC-RCCIIT/General-Purpose-Scripts/blob/main/README.md?plain=1#L43)

**Comment by OBITORASU** [comment]
Fixed that for him don't worry @belelaritra thanks for the heads up!

**Pull request merged**